src/main/java/org/jetbrains/research/anticopypaster/ide/ui/CustomMetricsMenu.java

```diff
@@ -0,0 +1,233 @@
+package org.jetbrains.research.anticopypaster.ide.ui;
+
+import com.intellij.openapi.components.PersistentStateComponent;
+import com.intellij.openapi.components.State;
+import com.intellij.openapi.components.Storage;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.project.ProjectManager;
+import com.intellij.openapi.ui.ComboBox;
+import com.intellij.openapi.ui.DialogWrapper;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.*;
+import java.util.Scanner;
+
+@State(name = "AntiCopyPasterUsageStatistics", storages = {@Storage("custom-anticopypaster-plugin.xml")})
```

Comment:
Is this stuff actually getting used? I remember we went to basic file writing and this just seems like extra stuff from development.